src/Bicep.Cli.Nuget/nuget.proj

```diff
@@ -0,0 +1,46 @@
+<!--
+  This project pulls down the setup compiler from a nuget and invokes it to build the setup exe.
+  It depends on the published self-contained bicep.exe and bicep.pdb to be placed in a bicep subfolder.
+  During the CI build, this happens via artifact download action.
+
+  Prerequisites:
+  - Copy bicep.exe and *.pdb from Bicep CLI publish directory to bicep\ directory under this project.
+-->
+<Project Sdk="Microsoft.Build.NoTargets">
```

Comment:
>NoTargets [](start = 30, length = 9)

This is a completely custom NuGet without a real assembly, so I had to do it myself in a NoTargets SDK project :( Thankfully NuGet.exe is available in a NuGet package and works on all the major OSs.